app/shared/resource-calendar/ResourceCalendar.js

```diff
@@ -128,13 +121,6 @@ export class UnconnectedResourceCalendar extends Component {
             </ResourceCalendarOverlay>
           </Overlay>
         </div>
-        <button
```

Comment:
Any reason for removing this?
This will make the date changing not working between the datepicker and the fullcalendar

Reply:
Those  were replaced with fullCalendars next and prev buttons so changing date is still possible.

Reply:
So basically no more date-picker?

Reply:
Please remove all the styling related to this also.

And remember, there are 2 place in the app that have FullCalendar

Reply:
Datepicker still exists and works, although its date isn't updated if date is  changed via fullcalendar buttons.

Removed styling for arrows that were removed.
FullCalendar is in two places but it seems that its using the same component so styles are in both of them.

Reply:
Nice!